client/signup/steps/rewind-form-creds/index.jsx

```diff
@@ -0,0 +1,71 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import Card from 'components/card';
+import Button from 'components/button';
+
+class RewindFormCreds extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+
+		// Connected props
+		siteSlug: PropTypes.string.isRequired,
+	};
+
+	stepContent = () => {
+		const { translate, siteSlug } = this.props;
+
+		return (
+			<Card className="rewind-form-creds__card rewind-switch__card rewind-switch__content">
+				<h3 className="rewind-form-creds__title rewind-switch__heading">
+					{ translate( 'Site credentials' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Add site credentials' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).